src/common/mutex_debug.h

```diff
@@ -198,6 +194,16 @@ class mutex_debug_impl : public mutex_debugging_base
     unlock_impl();
   }
 
+private:
+  bool _try_lock(bool no_lockdep = false) {
```

Comment:
Being an internal helper, does `_try_lock()` need to default the argument?
```suggestion
  bool _try_lock(bool no_lockdep) {
```

Reply:
done